inc/ZipProvider.php

```diff
@@ -119,22 +119,29 @@ public function generate_zip_file() : bool {
 		}
 
 		/* @var GP_Locale $locale */
-		$locale  = GP_Locales::by_slug( $this->translation_set->locale );
-		$project = GP::$project->get( $this->translation_set->project_id );
-		$entries = GP::$translation->for_export( $project, $this->translation_set, [ 'status' => 'current' ] );
+		$locale     = GP_Locales::by_slug( $this->translation_set->locale );
+		$gp_project = GP::$project->get( $this->translation_set->project_id );
+		$entries    = GP::$translation->for_export( $gp_project, $this->translation_set, [ 'status' => 'current' ] );
 
 		if ( ! $entries ) {
 			return false;
 		}
 
 		$files_for_zip = [];
+		$project       = new Project( $gp_project );
+		$text_domain   = $project->get_text_domain();
 
 		/* @var GP_Format $format */
 		foreach ( [ GP::$formats['po'], GP::$formats['mo'] ] as $format ) {
 			$file_name = str_replace( '.zip', '.' . $format->extension, $this->get_zip_filename() );
```

Comment:
It maybe better to define the `$file_name` outside the loop and add the extension within the loop.

Reply:
Mostly micro optimization at this point but I can change it 👍